abis_mapping/templates/survey_metadata/mapping.py

```diff
@@ -518,6 +519,10 @@ def add_spatial_coverage(
         graph.add((geometry_node, a, utils.namespaces.GEO.Geometry))
         graph.add((geometry_node, utils.namespaces.GEO.asWKT, geometry.to_transformed_crs_rdf_literal()))
 
+        if spatial_accuracy is not None:
+            accuracy = rdflib.Literal(spatial_accuracy, datatype=rdflib.XSD.double)
+            graph.add((geometry_node, utils.namespaces.GEO.hasMetricSpatialAccuracy, accuracy))
+
         self.add_geometry_supplied_as(
             subj=uri,
             pred=utils.namespaces.GEO.hasGeometry,
```

Comment:
will need to add arg `spatial_accuracy=accuracy` to this call i think and default `accuracy` to `None` i.e. `accuracy: rdflib.Literal | None = None`

Reply:
If accuracy isn't available, wouldn't you just not create the node?

Reply:
can't have it both ways :smile: ! If you're going to add the code for spatial accuracy since it doesn't exist as a field yet in the template, then you'll need to make sure the rest flows through, including a "geometry supplied as" node using the `add_geometry_supplied_as` method with the spatial accuracy provided to it.